src/v2/DevTools/createTestEnv.tsx

```diff
@@ -78,26 +78,6 @@ class TestEnv<MutationNames extends string, TestPage extends RootTestPage> {
       {} as any
     )
 
-    beforeEach(() => {
-      this.errors = []
-    })
-
-    afterEach(() => {
```

Comment:
not possible to use out of test files anymore, shifted the responsibility to the callers to control from the available methods